setup.py

```diff
@@ -30,7 +30,8 @@
 tests_require = ['pytest==3.3.1', 'pytest-xdist==1.22.1', 'python3-indy==1.6.8', 'pytest-asyncio==0.8.0']
 
 setup(
-    name=metadata['__title__'],
+    # TODO use __title__ from metadata instead
+    name='indy-node-dev',
```

Comment:
Why do we have `indy-node-dev` here again?

Reply:
Just a temporary fix. Will be improved in next coming PR.